README.md

```diff
@@ -0,0 +1,10 @@
+# Website for Nibble Computer Society
+
+Nibble Computer Society is the official computer society of JSS Noida. We are a group of students working towards enhancing knowledge in the field of computer science. The website shows our contribution to the college, our members and events & workshops that we organize.
+
+  Visit here : [hackncs.com](http://hackncs.com)
+
+  Follow us here :
+- [Facebook](https://www.facebook.com/nibblecomputersociety)
+- [Behance](https://www.behance.net/ncs-jss)
```

Comment:
This link is not opening.. Either fix it or remove it.

Reply:
Sir, this is an issue with Behance. It doesn't work on some connections